src/Icons/Link.tsx

```
@@ -0,0 +1,9 @@
+import IconType from "@/types/IconType";
+
+export default function LinkIcon({ width = 16, height = 16, color = "#fff" }: IconType) {
+  return (
+    <svg height={height} viewBox="0 0 16 16" width={width} fill={color}>
```

Biome

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)